credentials_manager/lib/credentials_manager/appfile_config.rb

```diff
@@ -41,12 +43,35 @@ def initialize(path = nil)
           # rubocop:disable Lint/Eval
           eval(content)
           # rubocop:enable Lint/Eval
+
+          print_debug_information(path: full_path) if $verbose
         end
       end
 
       fallback_to_default_values
     end
 
+    def print_debug_information(path: nil)
+      self.class.already_printed_debug_information ||= {}
+      return if self.class.already_printed_debug_information[self.data]
+      # self.class.already_printed_debug_information is a hash, we use to detect if we already printed this data
+      # this is necessary, as on the course of a fastlane run, the values might change, e.g. when using
+      # the `for_lane` keyword.
+
+      puts "Successfully loaded Appfile at path '#{path}'".yellow
+
+      self.data.each do |key, value|
+        puts "- #{key.to_s.cyan}: '#{value.to_s.green}'"
```

Comment:
Ah, the code is in _credentials_manager_, which has no dependency to `terminal-table`. I don't think it's worth adding the dependency for this